src/OSPSuite.Presentation/Presenters/Charts/ChartEditorPresenter.cs

```diff
@@ -434,6 +440,9 @@ private void updateUsedColumns()
       {
          if (Chart == null) return;
          _dataBrowserPresenter.InitializeIsUsedForDataColumns(Chart.UsedColumns);
+
+         //Once used columns are updated, we need to make sure we update the meta data related to used column as well
+         refreshColorGroupingPresenter();
```

Comment:
I have created a method here. When we changed the selected columns, we need to refresh somehow